tests/specs/General/Associations_Match.txt

```diff
@@ -0,0 +1,9 @@
+-- resources/test.txt --
+~~ lineWidth: 35, fold: fold -w {{line_width}} -s, fold.stdin: true, fold.associations: **/*.txt ~~
+== should do nothing when the association doesn't match ==
```

Comment:
nit: I think this test description does not match the test:
```suggestion
== should format when the association matches ==
```

Reply:
Thanks!